public/oniguruma/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+autoreconf -vfi
+./configure
```

Comment:
No —prefix=/usr ??

Reply:
What's the importance of it? I've seen other pkgs using it.

Reply:
![image](https://user-images.githubusercontent.com/7507990/61797900-5029fd80-adf6-11e9-8d53-2942c494d8e1.png)

Reply:
So the packages are installed to `/usr`, not `/usr/local`.

Reply:
Does KISS Linux have a preference?

Reply:
Yes. The preference is `--prefix=/usr`, so packages are installed to `/usr`, not `/usr/local`. Why? Because packages built from source *not* handled by package manager goes into `/usr/local`.

Reply:
@dylanaraps @konimex b34e468 adds `--prefix=/usr`. The package installs successfully.